openshift/templates/pims-api/deploy.yaml

```diff
@@ -18,34 +20,65 @@ objects:
   - kind: Route
     apiVersion: v1
     metadata:
-      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
```

Comment:
do we need this same string (${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}) associated to the route 4 times?